src/Internal.AspNetCore.Sdk/build/Common.props

```diff
@@ -22,6 +22,8 @@ Usage: this should be imported once via NuGet at the top of the file.
     <PackageProjectUrl>https://asp.net</PackageProjectUrl>
     <PackageRequireLicenseAcceptance>true</PackageRequireLicenseAcceptance>
     <Serviceable Condition="'$(Configuration)' == 'Release'">true</Serviceable>
+    <DebugType>Portable</DebugType>
```

Comment:
This is the default setting already.

https://github.com/dotnet/sdk/blob/cd25d6bbfc2aac61cc790c511900427ac64fdaa8/src/Tasks/Microsoft.NET.Build.Tasks/build/Microsoft.NET.Sdk.props#L78